### DISCUSSION TITLE
28. February 2023

### DISCUSSION THREAD

**Discussion by dimakuv** [gramineproject/gramine#1198] [Meeting notes]
### Agenda

*(please write your proposed agenda items in comments under this discussion)*

- [x] Woju: toolserver/VPS for packages and CI

#### Misc

- Dmitrii: we will periodically label and scrub GitHub issues.

- Dmitrii: GSC signing plugins -- what's the status?
  - Woju suggested a design that allows Jinja-Dockerfile-templates (as snippets, not as one big Dockerfile). This is a compromise between his proposed gramine-sgx-sign plugins (for core Gramine, but not suitable for GSC) and between Sankar's design with a `--template <single-big-dockerfile>`.
  - Woju: this design with snippets precludes users shooting themselves in the foot (e.g., forgetting to remove their keys from inside the Docker image).
  - Woju: `gsc sign-image` also needs smth like `--define variable=value` (like in gramine-manifest), that will replace Jinja-Dockerfile-templates' variables. For example, for our default local-key-file signing flow, we'll have `--define key=path/to/file`.

- Dmitrii: can we keep "Gramine features" in Markdown? and where do we put it (how people find it)?
  - Woju: yes. Michal: yes, but with a comment why we can't use RST.
  - Woju: we will need a Markdown renderer, see https://github.com/gramineproject/gramine/commit/f196e90ae3943ed76f768fb2e21f9d56bf59bfe1

#### Woju: toolserver/VPS for packages and CI

What is the budget for the Gramine project, from the perspective of the Linux Foundation?
- Intend to spend 1200 EUR for CI.
- TODO(woju): write to LF contacts.

**Comment by woju** [comment]
markdown renderer rip (to be reverted?): gramineproject/gramine@f196e90ae3943ed76f768fb2e21f9d56bf59bfe1
